kubernetes/samples/charts/voyager/README.md

```diff
@@ -118,6 +118,64 @@ $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:30305/testwebapp/
 $ curl -k -H 'host: domain2.org' https://${HOSTNAME}:30307/testwebapp/
 ```
 
+## SSL termination at ingress controller
+This sample demonstrates how to terminate SSL traffic at the ingress controller to access the WebLogic Server Administration Console through the SSL port. 
+
+### 1. Enable "WebLogic Plugin Enabled" on the WebLogic domain level
+
+If you are using WDT to configure the WebLogic domain, you need to add the following resource section at the domain level to the model YAML file.
+```
+resources:
+     WebAppContainer:
+         WeblogicPluginEnabled: true
+```
+If you are using a WLST script to configure the domain, then the following modifications are needed to the respective PY script.
+```
+# Configure the Administration Server
+cd('/Servers/AdminServer')
+set('WeblogicPluginEnabled',true)
+...
+cd('/Clusters/%s' % cluster_name)
+set('WeblogicPluginEnabled',true)
+```
+### 2. Update the frontendRules section Ingress resource.
```

Comment:
Please make ALL the same edits to this section of the doc, that I gave you for the same section in Traefik README. I do not want to re-print them all here but please make sure that you incorporate them all.

Reply:
Also modified the traefik README.md